extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -147,21 +147,19 @@ export abstract class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
+							const session = await this._postgresModel.controllerModel.acquireAzdataSession();
```

Comment:
This should actually be in its own outer try/catch - we want the save button to be re-enabled here if an error occurs during login as well

(same for all the other similar calls below)